Optimize RTE page (#398)
TYPO3-Documentation/TYPO3CMS-Reference-TSconfig: Documentation/PageTsconfig/Rte.rst

Please refer to the :ref:`RTE chapter <t3coreapi:rte>` in Core API document
for more general information on RTE configuration and data processing.

> The order in which configuration for the RTE is loaded is (the first one which

Reviewer: ```suggestion
The order in which the configuration for the RTE is loaded is (the first one which
```

> is set, is used, see :ref:`example <pageTsRteOverridePreset>` below):

Reviewer: ```suggestion
is set will be used, see :ref:`example <pageTsRteOverridePreset>` below):
```

> 1. preset defined for a specific field via PageTS

Reviewer: ```suggestion
1. preset defined for a specific field via page TSconfig
```

2. :ref:`richtextConfiguration <t3tca:columns-text-properties-richtextConfiguration>`
   defined for a specific field via TCA

> 3. general preset defined via PageTS (:typoscript:`RTE.default.preset`)

Reviewer: ```suggestion
3. general preset defined via page TSconfig (:typoscript:`RTE.default.preset`)
```

Configure RTE for a specific field in a table for a specific :ref:`record type <t3tca:types>`
    `RTE.config.[tableName].[fieldName].types.[type]`

Configuring RTE via page TSconfig is general and not specific to a

> specific RTE editor, however TYPO3 comes with EXT:rte_ckeditor, so this

Reviewer: ```suggestion
particular RTE editor, however TYPO3 comes with EXT:rte_ckeditor, so this
```

avoiding specific after specific in the same sentence.

> particular RTE editor, however TYPO3 comes with EXT:rte_ckeditor, so this

Reviewer: ```suggestion
particular rich-text editor, however TYPO3 comes with EXT:rte_ckeditor, so this
```

RTE editor -> E means already editor.

> particular rich-text editor, however TYPO3 comes with EXT:rte_ckeditor, so this

Reviewer: ```suggestion
particular rich-text editor. However, TYPO3 comes with EXT:rte_ckeditor, so this one
```

will usually be used. This page covers only the general configuration, for

> more information about configuring rte_ckeditor, see the

Reviewer: ```suggestion
more information about configuring EXT:rte_ckeditor, see the
```

   # But disable RTE for tt_content bodytext again if the record type is "text"
   RTE.config.tt_content.bodytext.types.text.disabled = 1

.. _pageTsRteOverridePreset:

Example: Override preset
------------------------

> Check the description of the order above to see which setting overrides which.

Reviewer: ```suggestion
Refer to the description of the order above for details of which setting has priority over which.
```

Summary:

> *   Setting the preset via page TSconfig *for a specific field* overrides all

Reviewer: I don't understand that sentence: "... overrides all else"

Author: If set, this is the setting which will be used, regardless of any other setting

Reviewer: Added a comma, to make it clearer.

> *   Setting the preset via page TSconfig *for a specific field* overrides all

Reviewer: ```suggestion
*   Setting the preset via page TSconfig *for a specific field* overrides all,
```
